docs/checks/internal/checks.yaml

```diff
@@ -199,6 +199,7 @@ checks:
       
       Tier 5 Requirements (10/10 points):
         - For administrators: Dismiss stale reviews
```

Comment:
Minor nit: can you format this so the list hierarchy is clear? Are these both under `For administrator`? If separate, put `dismiss stale review` under line 201 and indented. If both apply to admins, indent the `require CODEOWNER review` as well.

Reply:
Yes! Fixed.